.github/workflows/integration-test.yml

```diff
@@ -45,10 +43,48 @@ jobs:
         run: |
           printf "numba>=0.60.0\nnumpy<2.0" | tee /tmp/constraints.txt
           uv pip install --compile --system ".[test]" -c /tmp/constraints.txt -v 
-          uv pip install --compile --system git+https://github.com/scverse/anndata 
+          uv pip install --compile --system git+https://github.com/scverse/anndata
+        continue-on-error: true
+
+      - name: Set failure type for install
+        if: failure()
+        run: |
+          echo "Installation failed for ${{ matrix.package }}"
+          echo "failure_type=install" >> $GITHUB_ENV
+          echo "failure=true" >> $GITHUB_ENV
 
       - name: Env list
+        if: success()
         run: uv pip freeze
 
       - name: Run test
+        if: success()
         run: pytest
+        continue-on-error: true
+
+      - name: Set failure type for test
+        if: failure()
+        run: |
+          echo "Test failed for ${{ matrix.package }}"
+          echo "failure_type=test" >> $GITHUB_ENV
+          echo "failure=true" >> $GITHUB_ENV
+    
+      - name: Check if failure issue already exists
+        if: env.failure == 'true' # && github.event_name == 'schedule'
```

Comment:
here and in the next step: does this work, or do we need to prepend `failure() &&` to the condition?

GH actions is weird about conditions. I think the presence of `success()` or `failure()` changes how they are evaluated.